pkg/util/cluster/cluster.go

```diff
@@ -134,15 +134,38 @@ func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName str
 		return fmt.Errorf("fp service principal id is not found")
 	}
 
-	c.log.Infof("creating AAD application")
```

Comment:
The subscription I'm testing this on does not have Microsoft.Graph permissions so instead of creating a new AAD its just reusing RPs. This won't be part of the final PR for merge